content/v2.0/reference/config-options.md

```diff
@@ -1,193 +1,696 @@
 ---
 title: InfluxDB configuration options
 description: >
-  Configure InfluxDB when starting the `influxd` service.
-  This article outlines the different configuration options available.
+  Customize your InfluxDB configuration by passing configuration flags with the
+  [`influxd` daemon](/v2.0/reference/cli/influxd/), setting environment variables,
+  or defining configuration options in a configuration file.
 menu:
   v2_0_ref:
     name: Configuration options
     weight: 3
 products: [oss]
+related:
+  - /v2.0/reference/cli/influxd
 ---
 
+Customize your InfluxDB configuration by using configuration flags with the
+[`influxd` daemon](/v2.0/reference/cli/influxd/), setting environment variables,
+or defining configuration options in a configuration file.
+
+### Configuration precedence
+InfluxDB honors configurations settings using the following precedence:
+
+1. `influxd` daemon flags
```

Comment:
i mean, _technically_ `influxd` isn't a daemon unless you run it in the background. i feel like these are more commonly called command line flags or something